### PR TITLE
Milestone two

### PR DESCRIPTION
**Pull request by Arinpe**
This is a Bookstore application, where users get to display the books they have read or currently reading. This app is built with React and Redux

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { createStore } from 'redux';
+import { Provider } from 'react-redux';
+import App from './App';
+import rootReducer from './reducers';
+
+import './index.scss';
+
```
Comment (eliasecasta) [inline src/index.js:9]: - [ ] it adds example books to initial state object [screenshot from pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/InitialBooks.png) (You need to initialize your sample books here please)

File: src/App.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
```
Comment (eliasecasta) [inline src/App.js]: - [x] it is placed in `src/components` directory (Right now it is in src, please fix this to comply with the project requirements)

File: src/common.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  //   title: 'EAST OF EDEN',
+  //   author: 'JOHN STEINBECK',
+  //   category: categories[generateRandomNumber(categories.length)],
+  // },
+  // {
+  //   id: generateRandomNumber(),
+  //   title: 'THE SUN ALSO RISES',
+  //   author: 'ERNEST HEMINGWAY',
+  //   category: categories[generateRandomNumber(categories.length)],
+  // },
```
Comment (eliasecasta) [inline src/common.js]: - [x] Please remove all the commented code that you may have left.

File: src/actions/index.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
+import { ADD_BOOK, REMOVE_BOOK, FILTER_BOOK } from './type';
+
+export const addBook = (book) => ({
+  type: ADD_BOOK,
+  payload: book,
+});
+
+export const removeBook = (id) => ({
```
Comment (eliasecasta) [inline src/actions/index.js]: - Contains `CREATE_BOOK` and `REMOVE_BOOK` actions with Book objects as an argument [screenshot from pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone2/ReduxActions/ReducxActions.png) (Both actions should receive the book object as an argument, I understand that deletion by ID is not a bad idea but the project requires it like this, please change it)

**Review by eliasecasta — CHANGES_REQUESTED** [review]
# Status:  Changes Required ♻️
_Check the comments under the review_
<hr>

Hello @Arinpe, I am Elias, your reviewer for this PR

Well done so far with your project!

I will mention to you a small summary of things that need to be changed in order to keep moving:

- [x] I was unable to run your project successfully, this is the error that I am getting, please fix it to be able to display content on the page: 
image
- [ ] Please check the comments below for additional changes needed.

Please correct those issues and send another review request, good luck, and well done so far! 

Code review by Elias Castañeda

_(Feel free to leave any questions or comments in the PR thread if something is not 100% clear)_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: src/App.js [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+import BooksList from './containers/BooksList';
+import BookForm from './containers/BookForm';
+
+const App = () => (
+  <div className="App pb-5">
+    <BooksList />
+    <BookForm />
```
Comment (george-shammar) [inline src/App.js:8]: - [ ] As highlighted by the previous reviewer, the `App.js` component is still in the `src` folder. Please, implement it inside `src/component` as per the project requirement.

File: src/index.js [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+const store = createStore(
+  rootReducer,
+  // eslint-disable-next-line no-underscore-dangle
+  window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__(),
+);
+
+ReactDOM.render(
+  <Provider store={store}>
+    <React.StrictMode>
```
Comment (george-shammar) [inline src/index.js:18]: - [ ] As highlighted by the previous reviewer, please implement your sample book object here (index.js).

**Review by george-shammar — CHANGES_REQUESTED** [review]
Hi @Arinpe , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

- [ ] I couldn't run your application for the same reason mentioned by the previous reviewer. Please, let's fix it. :pray: 


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by rukundoeric — APPROVED** [review]
Hi @Arinpe,

_Excellent job on this project. Keep it up 🎆_

_Congratulations! 🎉  on the splendid job 👍
You can now merge this PR and submit the project completion form 🚀 . I wish you all the very best in your next tasks 🥇_

Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

------

**Pull request merged**